generate

```diff
@@ -10,6 +10,8 @@ PROTOCOL_RESILIENCE_ARGUMENTS='--version=3.8.1 --java --use=./ --data-plane'
 SWAGGER_PATH=node_modules/@microsoft.azure/autorest.testserver/swagger
 
 # Vanilla
+mv -f vanilla-tests/src/main/java/fixtures/report/CoverageReporter.java vanilla-tests/swagger/CoverageReporter.java
```

Comment:
Actually we seems not really need to run CoverageReport class. Just keep it there. This class is manual written.